src/Controls/samples/Controls.Sample.Profiling/Android/environment.emulator.txt

```diff
@@ -0,0 +1 @@
+DOTNET_DiagnosticPorts=10.0.2.2:9001,suspend
```

Comment:
Environment variables for the emulator. Because the emulator is already on the host, we don't need port forwarding and we can just use the special IP and the desired port.